tests/run_test.sh

```diff
@@ -100,6 +100,11 @@ cp ${PATHTR}/modulefiles/ufs_common*               ./modulefiles/.
 # Get the shell file that loads the "module" command and purges modules:
 cp ${PATHRT}/module-setup.sh                       module-setup.sh
 
+if [[ $MACHINE_ID == wcoss2.* ]] || [[ $MACHINE_ID == acorn.* ]] ; then
```

Comment:
I am curious why the current code does not need this? Anything has changed? I do notice issues with compare_ncfile.py when I test new libraries on acorn with ecflow. The compare_ncfile.py works in rt.sh when not running with ecflow

Reply:
The current code also needs this. So far all output files happen to bit identical on wcoss2 and we never need to run compare_ncfile.py. It works when rt.sh runs without ecflow because these modules are loaded in rt.sh, and are available when run_test.sh runs. But with ecflow run_test.sh runs on a different (compute) node as a separate process.